projects/cdk/schematics/ng-update/steps/remove-module.ts

```diff
@@ -29,6 +29,10 @@ export function removeModule(name: string, moduleSpecifier: string): void {
     const references = getNamedImportReferences(name, moduleSpecifier);
 
     references.forEach(ref => {
```

Comment:
what about
```ts
references.filter(ref => !ref.wasForgotten()).forEach(ref => {
```

Reply:
Or

```ts
    return arrayFlat(
        namedImports.map(specifier => specifier?.findReferencesAsNodes() || []),
    ).filter(ref => !ref.wasForgotten());
```

<img width="719" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/ec68c4a2-b7bd-422c-96ff-7a68737eeff0">

Reply:
1. It will iterate the same array twice.
2. We should check condition `wasForgotten()` before EVERY beginning of new iteration (because previous one can change everything).

Reply:
sorry